static/src/stylesheets/module/facia-garnett/_pillars.scss

```diff
@@ -62,10 +69,22 @@ $pillars: (
         headlineIcon: $lifestyle-garnett-main-1,
         metaText: $garnett-neutral-2,
         cutoutBackground: $lifestyle-garnett-feature,
-        invertedKicker: $lifestyle-garnett-media-main-1
+        invertedKicker: $lifestyle-garnett-media-main-1,
+        underline: $lifestyle-garnett-main-2
     )
 );
 
+@mixin garnett-underline($colour, $line-height) {
```

Comment:
Maybe we could call this mixin `analysis-underline`, since this is the only card type that receives this styling

Reply:
I thought about it but chose a neutral word in case we use it somewhere else. I don't mind changing if you all think it's best 😃